test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.conditional.test.ts

```diff
@@ -24,10 +24,10 @@ describe('parse bpmn as json for conditional sequence flow', () => {
   each([
     ['exclusiveGateway', SequenceFlowKind.CONDITIONAL_FROM_GATEWAY],
     ['inclusiveGateway', SequenceFlowKind.CONDITIONAL_FROM_GATEWAY],
+    ['callActivity', SequenceFlowKind.CONDITIONAL_FROM_ACTIVITY],
```

Comment:
:question: why changing the order?

Reply:
I added some scpecific tests for call Activity, and after remove them in order to comme back to the previous tests, but I didn't remember the order 😉